modules/dnn/perf/perf_net.cpp

```diff
@@ -265,7 +240,7 @@ PERF_TEST_P_(DNNTestNetwork, FastNeuralStyle_eccv16)
 {
     if (backend == DNN_BACKEND_HALIDE)
         throw SkipTestException("");
-    processNet("dnn/fast_neural_style_eccv16_starry_night.t7", "", "", Mat(cv::Size(320, 240), CV_32FC3));
+    processNet("mosaic-9.onnx", "", "", Mat(cv::Size(224, 224), CV_32FC3));
```

Comment:
I suspect that the path should be `dnn/mosaic-9.onnx`

Reply:
So is this `mosaic-9.onnx` the missing model?

Reply:
Please change the file path according to https://github.com/opencv/opencv_extra/pull/1097/files#r1357831844. I will upload the model to the target place.